ext/ageobw/eu.esdihumboldt.hale.app.bgis.ade/src/eu/esdihumboldt/hale/app/bgis/ade/defaults/GenerateDefaults.java

```diff
@@ -55,6 +55,8 @@
 
 	private Schema schema;
 
+	private String schameResourceId;
```

Comment:
`schemaResourceId` ?

Reply:
Sorry for the typo. Now refactored to resourceId.